nuclear/message/generator/Enum.py

```diff
@@ -175,9 +175,15 @@ def generate_cpp(self):
                 return static_cast<{protobuf_name}>(value);
             }}
 
-            std::ostream& {namespace}::operator<< (std::ostream& out, const {fqn}& val) {{
-                return out << static_cast<std::string>(val);
-            }}"""
+            namespace {namespace} {{
+
+                std::ostream& operator<< (std::ostream& out, const {fqn}& val) {{
+                    return out << static_cast<std::string>(val);
+                }}
+            
+            }}
+
+            """
```

Comment:
What did this do?

Reply:
The code this generated created a warning that the operator was not declared in the namespace, and that it was only there as a friend. This removed that warning. Although I am not exactly sure why this is different to the compiler than 
`std::ostream& {namespace}operator<< (std ...)`
but it got rid of it.

Reply:
Interesting... I'm glad you found a way to get rid of that warning I couldn't find one.
You'll need to make sure that printing out enums still works.
Remove the excess whitespace around the namespace here as well.